src/MidiPlayer.js

```diff
@@ -624,4 +625,36 @@ not.
         this.logging = logging;
         this.eventHandler.setLogger({ eventLogger, logging });
     }
+    /*
+     * get total duration time
+     * 2021 03 NormalUniversity
+     */
+    getDuration() {
+
+        return this.totalTime;
+    }
+    /**
+     * Seek Position.
+     * @param {any} [mSecond] milli seconds to be seek
+     * @example
+     *  let midiPlayer = new MidiPlayer({ logging: false, patchUrl: 'lib/pat/' });
+     *  midiPlayer.play({url: $url})
+     *  midiPlayer.seek({mSecond: $milliSeconds});
+     */
+    async seek({ mSecond}) {
```

Comment:
To fit with the pattern used by other methods, I would suggest using the name `millisecond` as a parameter instead of the abbreviated `mSecond`. Furthermore, "millisecond" is a single word. It is not spelled as "milli second" so there is no need to turn the "s" of "second" into "S".